addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1027,6 +1030,61 @@ options.registry.anchor = options.Class.extend({
     },
 });
 
+options.registry.CookiesBar = options.registry.SnippetPopup.extend({
+    xmlDependencies: [
+        '/website/static/src/xml/website.cookies_bar.xml',
+    ],
```

Comment:
One line :+1: SnippetPopup does not have xmlDependencies? Could be worth extending it anyway to be safe.

Reply:
No, but better safe than sorry, addes the concat